sites/postcss-preset-env/blog/postcss-color-mix-function.md

```diff
@@ -148,12 +201,93 @@ After `color-mix()` we will focus on bringing relative color syntax to PostCSS b
 		color: white;
 		display: block;
 		font-size: 0.875em;
+		font-family: monospace;
 		line-height: 2;
 		margin: 1rem auto;
-		max-width: calc(100% - 3rem);
+		max-width: 100%;
 		padding: 2px 8px;
 		position: relative;
 		text-align: left;
-		width: 400px;
+		width: 525px;
+	}
+
+	input[type="color"] {
+		-moz-appearance: none;
+		-webkit-appearance: none;
+		appearance: none;
+		background-color: transparent;
+		border: none;
+		cursor: pointer;
+		height: 50px;
+		transition: opacity .3s;
+		margin: 0;
+		width: 50px;
+	}
+	
+	input[type="color"]::-webkit-color-swatch {
+		border-radius: 50%;
+		border: 2px solid #000000;
+	}
+	
+	input[type="color"]::-moz-color-swatch {
+		border-radius: 50%;
+		border: 2px solid #000000;
+	}
+
+	input[type="color"]:is(:hover,:focus) {
+		opacity: 0.8;
+	}
+
+	.color-mix-settings__selects {
+		display:flex;
+	}
+
+	.color-mix-settings__select {
+		display: flex;
+		flex: 1;
+		flex-direction: column;
+		gap: 4px;
+	}
+
+	select {
+		background-image: url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' fill='none' viewBox='0 0 20 20'%3E%3Cpath stroke='%236B7280' stroke-linecap='round' stroke-linejoin='round' stroke-width='1.5' d='m6 8 4 4 4-4'/%3E%3C/svg%3E");
+		background-position: right 0.5rem center;
+    background-repeat: no-repeat;
+    background-size: 1.5em 1.5em;
+		font-size: 1rem;
+		color: #333;
+		padding: 10px;
+		border: 2px solid #ccc;
+		border-radius: 5px;
+		background-color: #fff;
+		appearance: none;
+		-webkit-appearance: none;
+		-moz-appearance: none;
+		margin: 0;
+		transition: border .3s;
+	}
+
+	select:hover {
+		border-color: #666;
+	}
+
+	input[type=range] {
+		-webkit-appearance: none;
+		width: 100%;
+		height: 10px;
+		border-radius: 5px;
+		background-color: #ddd;
+		margin-inline: 0;
+		outline: none;
+	}
+	
+	input[type=range]::-webkit-slider-thumb {
+		-webkit-appearance: none;
+		appearance: none;
+		width: 20px;
+		height: 20px;
+		border-radius: 50%;
+		background-color: #0077ff;
```

Comment:
Can we make this `csstools` branded green?

Reply:
Nice idea!